linkml-schema/library_generation.yaml

```diff
@@ -15,31 +19,34 @@ default_range: string
 subsets:
   bican:
     description: A subset of classes that are associated with BICAN.
-    from_schema: https://identifiers.org/brain-bican/kb-model
+    from_schema: https://identifiers.org/brain-bican/library-generation-schema
```

Comment:
what's `from_schema` ?

Reply:
this is an [additional id](https://linkml.io/linkml-model/latest/docs/from_schema/) that is added by schemasheet
but seems like the error came from the subset that were added from biolink_bican (I've added `imports` to be added automatically by schemasheet). For now I will just remove it, and add to my script, and after workshop I can check it